sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/StorageApiSinkSchemaUpdateIT.java

```diff
@@ -539,4 +560,151 @@ public void testAtLeastOnceWithIgnoreUnknownValues() throws Exception {
   public void testAtLeastOnceWithAutoSchemaUpdate() throws Exception {
     runStreamingPipelineWithSchemaChange(Write.Method.STORAGE_API_AT_LEAST_ONCE, true, true);
   }
+
+  public void runDynamicDestinationsWithAutoSchemaUpdate(boolean useAtLeastOnce) throws Exception {
```

Comment:
are these new tests intended to run on direct runner (GCP IO PreCommit) or also on Dataflow (Dataflow Runner v1/v2 PostCommit) suite? If the latter, please trigger a PostCommit job to check if the test setting works on a remote runner. If the former, add these tests in the exclusion test in https://github.com/apache/beam/blob/97c4134bfc311093e947abde68eddcd51fce375b/runners/google-cloud-dataflow-java/build.gradle#L631 and https://github.com/apache/beam/blob/97c4134bfc311093e947abde68eddcd51fce375b/runners/google-cloud-dataflow-java/build.gradle#L682

Reply:
Yep this should add two test runs to the Dataflow runner. triggering postcommits now